packages/ui/src/components/smui-wrapper/custom-drawer/custom-drawer.svelte

```diff
@@ -0,0 +1,80 @@
+<div id="custom-drawer">
+	<Drawer variant="dismissible" bind:open={$open}>
```

Comment:
Clicking outside the drawer area closes the drawer, is this by design?

Reply:
I change the behavior, and adapt reactivity see 584f1fc